src/libraries/System.Drawing.Common/src/System/Drawing/Drawing2D/GraphicsPath.Unix.cs

```diff
@@ -901,5 +901,19 @@ public void Widen(Pen pen, Matrix? matrix, float flatness)
             int s = Gdip.GdipWidenPath(_nativePath, pen.NativePen, m, flatness);
             Gdip.CheckStatus(s);
         }
+
+        /// <summary>
+        /// The underlying graphics path handle
+        /// </summary>
+        public IntPtr Handle => _nativePath;
```

Comment:
We try to order fields then properties then methods, with public before private. I'm surprised the editorconfig isn't flagging this for you.